src/bacnet/basic/object/csv.c

```diff
@@ -439,22 +423,18 @@ static char *CharacterString_Value_Description(uint32_t object_instance)
  * @return True on success, false otherwise.
  */
 bool CharacterString_Value_Description_Set(
-    uint32_t object_instance, BACNET_CHARACTER_STRING *new_descr)
+    uint32_t object_instance, const char *new_descr)
```

Comment:
`const char *const` ?? 😋

Reply:
Lets be bold const char * const restrict new_descr; ;)